client-registry-santempi/docker-compose-postgres.yml

```diff
@@ -12,6 +12,7 @@ services:
       REPMGR_NODE_NAME: santempi-psql-1
       REPMGR_PRIMARY_HOST: ${SANTEMPI_REPMGR_PRIMARY_HOST}
       REPMGR_PARTNER_NODES: ${SANTEMPI_REPMGR_PARTNER_NODES}
+      POSTGRESQL_POSTGRES_PASSWORD: postgres
```

Comment:
We should rather have this configurable via an env var and defaulted to this value in the `package-metadata.json` file as I'm sure people won't want to keep this default as their password in production.